### DISCUSSION TITLE
What effect does the gravity vector have on characters?

### DISCUSSION THREAD

**Discussion by jankrassnigg** [jrouwe/JoltPhysics#210] [Q&A]
CharacterVirtual::Update() takes a gravity vector as input and it is somehow used to solve constraints. It does not, however, have any effect of pulling the character downwards, as far as I can tell. The documentation doesn't mention what it is for.

I also didn't notice any difference with the vector set to -1000 or 0. What difference should I look out for? Also, does it need to be the same value, as what I otherwise use to manually pull the character down, or does it simply need the general direction?

**Comment by jrouwe** [comment]
Gravity is currently only used to calculate the impulse a dynamic object under the character receives when the character interacts with it. So it doesn't need to be the same gravity that you applied to the linear velocity of the character before the update.
